tests/unit/button/button_events.js

```diff
@@ -3,24 +3,16 @@
  */
 (function($) {
 
-module("button: events");
+module( "Button: events" );
 
-test("buttonset works with single-quote named elements (#7505)", function() {
-	expect( 1 );
-	$("#radio3").buttonset();
-	$("#radio33").click( function(){
-		ok( true, "button clicks work with single-quote named elements" );
-	}).click();
-});
-
-asyncTest( "when button loses focus, ensure active state is removed (#8559)", function() {
+asyncTest( "When button loses focus, ensure active state is removed", function() {
```

Comment:
This is a weird test. The bound and triggered events seem pretty random (aka magic). Should at least add an assertion that checks that the active class is there at one point, otherwise it might just be verifying that nothing at all happens.